drivers/include/periph/rtc.h

```diff
@@ -131,6 +131,20 @@ void rtc_poweroff(void);
  */
 void rtc_tm_normalize(struct tm *time);
 
+/**
+ * @brief Compare two time structs.
+ *
+ * @note  The time structs are assumed to be normalized.
```

Comment:
```suggestion
 * @pre   The time structs in @p a and @p b are normalized.
```

Also you could add an `@see rtc_tm_normalize`